### PR TITLE
Add update provisioning profile specifiers helper

### PR DESCRIPTION
**Pull request by slightair**
for Xcode8

### CODE REVIEW

File: lib/dokumi/tool/xcode/project_helper.rb [hunk 1]
```diff
@@ -108,6 +108,18 @@ def update_provisioning_profiles(provisioning_profiles)
           @xcodeproj.save
         end
 
+        # Set the provisioning profile specifiers for multiple targets. The key is the target's' bundle identifier.
```
Comment (vincentisambart) [inline lib/dokumi/tool/xcode/project_helper.rb]: s/target's'/target's/

File: lib/dokumi/tool/xcode/project_helper.rb [hunk 2]
```diff
@@ -108,6 +108,18 @@ def update_provisioning_profiles(provisioning_profiles)
...
+        def update_provisioning_profile_specifiers(provisioning_profile_specifiers)
+          each_build_settings do |build_settings|
+            bundle_identifier = build_settings["PRODUCT_BUNDLE_IDENTIFIER"]
+            next unless bundle_identifier
+            provisioning_profile_specifier = provisioning_profile_specifiers[bundle_identifier]
+            next unless provisioning_profile_specifier
+            build_settings["PROVISIONING_PROFILE_SPECIFIER"] = provisioning_profile_specifier
```
Comment (vincentisambart) [inline lib/dokumi/tool/xcode/project_helper.rb:121]: You should also remove existing provisioning profile specifiers as done in `update_provisioning_profiles`.

``` ruby
build_settings.keys.each do |setting_name|
  build_settings.delete(setting_name) if setting_name.start_with?("PROVISIONING_PROFILE_SPECIFIER[sdk=")
end
build_settings["PROVISIONING_PROFILE_SPECIFIER"] = provisioning_profile_specifier
```

**Review by vincentisambart — CHANGES_REQUESTED** [review]

File: lib/dokumi/tool/xcode/project_helper.rb [hunk 3]
```diff
@@ -108,6 +108,21 @@ def update_provisioning_profiles(provisioning_profiles)
           @xcodeproj.save
         end
 
+        # Set the provisioning profile specifiers for multiple targets. The key is the target's bundle identifier.
+        def update_provisioning_profile_specifiers(provisioning_profile_specifiers)
+          each_build_settings do |build_settings|
+            bundle_identifier = build_settings["PRODUCT_BUNDLE_IDENTIFIER"]
+            build_settings.keys.each do |setting_name|
+              build_settings.delete(setting_name) if setting_name.start_with?("PROVISIONING_PROFILE_SPECIFIER[sdk=")
+            end
```
Comment (vincentisambart) [inline lib/dokumi/tool/xcode/project_helper.rb]: Wouldn't it be better to do that after `next unless provisioning_profile_specifier`?

Comment (slightair) [reply lib/dokumi/tool/xcode/project_helper.rb]: 35a1a900

**Review by vincentisambart — APPROVED** [review]
LGTM 🚀

**Comment by slightair** [comment]
Thanks 😸

**Pull request merged**